### ISSUE TITLE
Header params as tag values

### ISSUE THREAD

**Issue by ellisjoe** [palantir/tritium#48]
Header params are separated by commas (https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2) which conflicts with our tagged key/value pair delimiter.

**Comment by schlosna** [comment]
Yeah, thinking we might want to use ASCII character codes 29, 30, 31 as separators, or go full structured serDe like Jackson. Thoughts?

Will also add tests for `user-agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36]`
